### ISSUE TITLE
Fix cell toolbar layout

### ISSUE THREAD

**Issue by steff456** [Quansight-Labs/jupyterlab-accessible-themes#35]
Currently in jupyterlab when the main content area doesn't have space the cell layout breaks as shown in the image below,

![image](https://user-images.githubusercontent.com/20992645/188001953-6e97098e-dd39-43dc-8c79-cc6ae43b4fd4.png)

Our goal is to preserve the horizontal layout as it will normally be in jupyterlab or google collab,

## Jupyterlab

<img width="1552" alt="image" src="https://user-images.githubusercontent.com/20992645/188003845-757acdd1-47bf-46f9-92eb-6a0e025e84c7.png">

## Google collab

<img width="1652" alt="image" src="https://user-images.githubusercontent.com/20992645/188003585-91a56157-3886-4182-bda8-9ae591c90cd7.png">

The first steps for solving this issue are,

1. Identify the component that is making the layout break - I will suspect it is the container for the toolbar
2. Check if there's some css magic that could fix it - This can be done by modifying the css in the browser
3. Identify the code that needs to be changed inside the project

After this is done we can check in and see where the change is needed and we can also discuss if we will attempt to add this change to the core of jupyterlab or the extension.
